css/templates/gallery/page-vars.css

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ :root {
+  --font-size: 16;
+  --font-color-primary: #000;
+  --font-family-primary: "Poppins", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", sans-serif;
+  --font-family-secondary: "Oswald", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", sans-serif;
+  --color-primary: #3F4AE6;
```

Comment:
nit: lowerc case hex values (please change in next PR)

Reply:
No problem 👍